pandokia/pcgi_treewalk.py

```diff
@@ -198,7 +209,7 @@ def treewalk():
             lquery[var] = '*'
             header_table.set_value(row, 0, label)
             header_table.set_value(row, 1, '=')
-            header_table.set_value(row, 2, escape(lquery[var]))
+            header_table.set_value(row, 2, escape(query[var]))
```

Comment:
It was a bug introduced before I modified this file. I saw the glitch Pandokia test result page only show host or context as * on the header no matter which context or host you are browsing (and in Pandokia production it shows those info), so I compare with the Pandokia production code and found out the original code has `query` instead of `lquery`. Notice that on line 209, `lquery[var]` is set to * and that's why the host or context was not showing properly in glitch Pandokia.